src/main/java/org/geppetto/simulation/manager/GeppettoManager.java

```diff
@@ -89,8 +89,14 @@ public GeppettoManager(IGeppettoManager manager)
 		super();
 		if(manager instanceof GeppettoManager)
 		{
+			IUser u = manager.getUser();
```

Comment:
@jrmartin those instructions seem to be left over from some debugging, can we delete them?

Reply:
@tarelli Debugging lines removed